octoprint_mrbeam/analytics/usage_handler.py

```diff
@@ -466,8 +467,8 @@ def write_usage_analytics(self, action=None):
         try:
             usage_data = dict(
                 total=self._usage_data[self.TOTAL_KEY][self.JOB_TIME_KEY],
-                prefilter=self.get_prefilter_usage(),
-                carbon_filter=self.get_carbon_filter_usage,
+                prefilter=self._get_prefilter_usage_time(),
+                carbon_filter=self._get_carbon_filter_usage_time,
```

Comment:
I think parenthesis are missing
```suggestion
                carbon_filter=self._get_carbon_filter_usage_time(),
```